packages/ember-metal/lib/keys.js

```diff
@@ -7,7 +7,7 @@
   @for Ember
   @param {Object} obj
   @return {Array} Array containing keys of obj
-  @private
+  @public
```

Comment:
@rwjblue Should I change the PR to remove the file, or should I change it to just deal with `Ember.compare`?